packages/react-components/react-breadcrumb-preview/stories/Breadcrumb/BreadcrumbBestPractices.md

```diff
@@ -7,13 +7,17 @@
 
 - Place Breadcrumbs at the top of a page, above a list of items, or above the main content of a page.
 - Provide label to the Breadcrumb component using `aria-label` or `aria-labelledby` prop.
-- `aria-current="page"` should be used in the current BreadcrumbItem, to indicate which page is currently displayed. If the BreadcrumbItem representing the current page is not a link, `aria-current` is optional.
+- Set the `current` prop on the last item.
+- Use `aria-current="page"` in the current BreadcrumbButton, to indicate which page is currently displayed.
+- For BreadcrumbItem `aria-current="page"` is optional because it's a non-interactive item.
```

Comment:
The guidance about `aria-current="page"` is a little hard to figure out:
* The last item is never supposed to be interactive, but `aria-current="page"` is optional for non-interactive items... so it's unclear when we should be setting it?
* None of the examples have this set -- can you add it in the appropriate places so the examples reflect the recommended behavior?
* Or, can BreadcrumbItem/Button just add it automatically when `current` is set instead of requiring the user to set it? This would be ideal if possible -- our components should be accessible by default.

Reply:
BreadcrumbButton is already have it when `current` is set. So I'll just remove this part